src/inverseRobot.js

```diff
@@ -4,10 +4,30 @@
  * @param {object} robot
  *
  * @return {object}
-*/
+ */
 
 function inverseRobot(robot) {
   // write code here
+  const values = Object.values(robot);
+
+  const newRobot = {};
+
+  for (let i = 0; i < values.length; i++) {
+    for (let j = i + 1; j < values.length; j++) {
+      if (values[i] === values[j]) {
+        return null;
+      }
+    }
+  }
+
+  Object.entries(robot).forEach((e) => {
```

Comment:
Do you really need to iterate all the properties again? Can't you do everything with only one loop?